.changeset/selfish-lizards-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-analytics-module-ga': minor
```

Comment:
Oh!  One other quick note: while this package is still `0.x`, changes like this (which are backwards compatible) should be marked `patch`.

Could be nice to link to the new section in the README in the changeset below too.  In a `for more information on how...` kind of way.

Reply:
Changed to `patch` and added a link